optuna_dashboard/ts/apiClient.ts

```diff
@@ -83,6 +83,30 @@ const convertPreferenceHistory = (
   }
 }
 
+interface FeedbackComponentResponse {
+  type: string
+  artifact_key?: string
+}
+
+const convertFeedbackComponentType = (
+  res?: FeedbackComponentResponse
+): FeedbackComponentType => {
+  if (res === undefined) {
+    return {
+      output_type: "note",
+    } as FeedbackComponentNote
+  }
+  if (res.type === "artifact") {
+    return {
+      output_type: "artifact",
+      artifact_key: res.artifact_key,
+    } as FeedbackComponentArtifact
+  }
+  return {
+    output_type: "note",
+  } as FeedbackComponentNote
+}
```

Comment:
A nit, but I think it's easier to update Python API response schema than creating this convert function.